homeassistant/components/motioneye/config_flow.py

```diff
@@ -72,15 +72,15 @@ def _get_form(
                         ): str,
                         vol.Optional(
                             CONF_ADMIN_PASSWORD,
-                            default=user_input.get(CONF_ADMIN_PASSWORD),
+                            default=user_input.get(CONF_ADMIN_PASSWORD, ""),
                         ): str,
```

Comment:
This seems like an odd combination. It is optional, but defaults to "", so it isn't optional?

Reply:
To step back: The issue here is the frontend refuses an empty password. Everything upstream of this (the client library, and motionEye itself) works fine without a password, and there have been [complaints](https://github.com/home-assistant/core/issues/75696) from people who like to run password-less.

Perhaps you'd prefer the field listed as `Required` but with the default, or is a better way to express to the frontend that it should not truly complain when the field is empty?

Yes, this is tested with and without passwords.

The problem without this PR:

![motioneye](https://user-images.githubusercontent.com/1136829/212488289-1c1cb5d9-aded-4963-b1e1-ddd29e23435c.png)

Reply:
> Perhaps you'd prefer the field listed as Required but with the default

That won't work, as it wouldn't allow an empty value. Suggested value is another option that could be used. However, that will alow just unset it to `None` and not `""`, I guess the provided solution starts to make sense now 👍 

Thanks @dermotduffy 👍